scribblings/quickscript.scrbl

```diff
@@ -59,14 +59,14 @@ can add your own scripts there too if you think they may be useful to others.
 
 @section{Make your own script: First simple example}
 
-Click on the @gui{Scripts|Manage scripts|New script...} menu item, and enter @gui{Reverse} for the script name.
+Click on the @gui{Scripts|Manage|New script...} menu item, and enter @gui{Reverse} for the script name.
 This creates and opens the file reverse.rkt in the user's scripts directory.
 Also, a new item automatically appears in the @gui{Scripts} menu.
 
 In the .rkt file that just opened in DrRacket, modify the @racket[define-script] definition to the following:
 @margin-note{Don't name your script function @racket[reverse], it would shadow Racket's own and make the script hang.}
 @margin-note{If you later change the @racket[#:label] property, you will need to reload the menu by clicking on
-@gui{Scripts|Manage scripts|Reload scripts menu} after saving the file).}
+@gui{Scripts|Manage|Reload scripts menu} after saving the file).}
```

Comment:
This should be `Scripts|Manage|Reload menu`

Reply:
done